README.md

```diff
@@ -59,7 +59,18 @@ Note how the cucumber-tsflow Decorators are being used to bind the methods in th
 
 ###### Compiling your TypeScript Support Code
 
-You'll also need a `tsconfig.json` file to compile your code. You'll also need to ensure that the `"moduleResolution": "node"` compiler option is set in order to bring in the typings that are shipped with cucumber-tsflow.
+You'll also need a `tsconfig.json` file to compile your code. The minimum requirements are:
```

Comment:
From the tinkering I've been doing over the last couple of days I think the module is important, at least. "umd" and "commonjs" will work but "es6" will make everything break as the export assignment "export = MySteps" is not permitted:

`error TS1203: Export assignment cannot be used when targeting ECMAScript 2015 modules. Consider using 'export default' or another module format instead.`

CucumberJS doesn't like es6 modules either and will complain about the use of import if it makes it into the transpiled code.